cli/src/funTest/assets/semver4j-ort-result.yml

```diff
@@ -318,6 +318,7 @@ scanner:
           path: "LICENSE.md"
           start_line: 3
           end_line: 3
+  files: []
```

Comment:
Maybe a stupid question, but why is this list empty? Given that we have findings above, shouldn't this list at least contain the files with findings? Similar in the other result files.

Reply:
This commits only adds the model classes, but the scanner outputs only empty file lists still. This is the next. The next next commit changes the scanner to add actual data. 

In that commit I added only the file lists needed for testing them, but I haven't added them to all assets, as these aren't used by the tests anyway. As adding could be quite a bit of effort, I thought it's better to do this only when need arises, and focus only on the test asset use for testing these file lists.

Reply:
I get your point that it's effort, but I'm unsure whether it's ok to have test results with inconsistent / incomplete content... could your new helper-cli command to rewrite test assets become handy here maybe?

Reply:
Do you agree that adding the data would go into the topmost commit of this PR?

Reply:
Ok, I've updated all the assets to contain file list entries as you proposed. 
I've decided to make this a separate commit on top, so that the commit which implements returning the file lists in the scanner still nicely illustrates the impact: turning a previously empty list into a properly filled one.

Reply:
Thanks!